source/sass/components/take-action.scss

```diff
@@ -0,0 +1,17 @@
+.take-action-section {
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.take-action-title {
+  font-family: $font-family-sans-serif;
+  font-size: 23px;
+  font-weight: bold;
+  line-height: 1.3;
+  color: $black;
+  margin-bottom: 27px;
+}
+
+.take-action-link:hover {
+  text-decoration: none;
```

Comment:
This is not needed. Tais mentioned to mimic card design on the /blog page: https://github.com/mozilla/foundation.mozilla.org/issues/4942#issuecomment-666739257